UI.AutomationTests/CommonUiTest.cs

```diff
@@ -135,6 +135,11 @@ await TestContext.Out.WriteLineAsync(e.StatusCode == (int)HttpStatusCode.NotFoun
 
     private async Task DeleteHearings()
     {
+        if (EnvConfigSettings.Environment.Equals("prod", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
could we make this a constant?

maybe add this to EnvironmentConfigSettings:

```
public static class Environments
{
    public const string Prod = "prod";
}
```

Reply:
will do!